charts/camunda-platform-alpha/test/integration/testsuites/vars/files/testsuite-core.yaml

```diff
@@ -305,3 +311,67 @@ testcases:
 #     - result.body ShouldContainSubstring optimize
 #     - result.body ShouldContainSubstring web-modeler-restapi
 #     - result.body ShouldContainSubstring core
+#     - result.body ShouldContainSubstring zeebe
+#     - result.body ShouldContainSubstring zeebe-gateway
+
+  ########################################################Port forwarding Camunda components##################################
+- name: Port forwarding Camunda components
+  steps:
+  - name: "Port forward to {{ .value.component }} service "
+    vars:
+      portforward: kubectl port-forward svc
+    type: exec
+    range:
+    - component: zeebe-gateway
+      script: "{{.portforward}}/camunda-zeebe-gateway 26500:26500" 
```

Comment:
@mabdelgowa, this will not work as port forwarding is a blocking command, and it will not return (will wait forever as it works).

So you can port forward and check at the same time in the `script` section something like this:

```
[...]
 range:
  - component: "Zeebe Gateway"
    serviceName: "camunda-zeebe-gateway"
    servicePort:  "26500"
    [...]
script: |
  (kubectl port-forward svc/{{ .serviceName }} {{ .servicePort }}:{{ .servicePort }} &
  curl localhost:{{ .servicePort }})
```

Please ensure to test it locally first to ensure the proposed change works as expected (you can test it with any service, not necessarily Camunda deployment).